aeon_sql.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+import "aeon_common.proto";
+
+package aeon;
+
+option go_package = "./pb";
```

Comment:
Shouldn't we use a proper name here? Take a look at [YDB](https://github.com/ydb-platform/ydb-api-protos/blob/1440f00ea8c997bfe378c9e8e8a29c8916b8b4ba/ydb_monitoring_v1.proto#L5), for example.

Reply:
Done.